checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -0,0 +1,63 @@
+# Enable StrictMode 3.0
+Set-StrictMode -Version 3.0
+
+class Config {
+    [bool]$rcc
+
+    Config() {
+        # Default constructor
```

Comment:
Is this mandatory in Powershell? Otherwise, one would of course prefer a "standard" constructor which accepts rcc as a parameter.

Reply:
No, it's not. We can avoid the default constructor and Powershell will on it's own assign the value `$false` to the rcc parameter.